pages/departure_page.py

```diff
@@ -10,7 +13,45 @@ def should_be_departure_page(self):
         print('Current url departure', current_url_departure)
         assert 'departure' in current_url_departure, f"This is not departure page, expected 'departure' in the {current_url_departure}"
 
+    def get_header(self):
+        departure_header = self.is_element_present(*DeparturePageLocators.DEPARTURE_HEADER)
+        assert departure_header, 'This is not a header in departure page'
+        departure_header_text = self.browser.find_element(*DeparturePageLocators.DEPARTURE_HEADER).text
+        return departure_header_text
+
+    def should_be_departure_header(self, departure_header):
+        print('Departure header: \n', departure_header)
+        expected_text = "Летим Из Москвы"
+        print('Expected header: \n', expected_text)
+        self.should_be_exact_text(expected_text, departure_header)
+
+    def get_description(self):
```

Comment:
```suggestion
    def get_departure_description_text(self):
```
Иначе непонято, что за description. Может это description карточек